### ISSUE TITLE
Feature Request: Cycle on/off status at custom interval

### ISSUE THREAD

**Issue by notcammy** [ashvin-bhuttoo/PersistentHotspot#2]
I found that after a while, devices won't connect properly to the hotspot anymore unless I turn it back off and on again. I'd like to see a feature implemented where it (optionally) automatically toggles the hotspot status at a custom interval (every hour for example).

Although this might just be an issue on my machine only others may experience the same problem and this would be a viable workaround for that.

If you don't have the time I'm willing to make a pull request if you want. 

Thanks!

**Comment by ashvin-bhuttoo** [comment]
Does the tray icon turn yellow or red when that happens? How do you turn it back off and on again, using the tray icon or other means? Yea sure i can add this feature for you :)

**Comment by notcammy** [comment]
It stays green. I toggle it off and on again with the tray icon (assuming that just does the same thing as manually toggling it in Windows settings) and it'll work fine again.

**Comment by ashvin-bhuttoo** [comment]
Have you tried updating your wifi driver? Are you using external usb wifi adapter?

**Comment by notcammy** [comment]
It's an old PCI Express card that has been lying around for a decade, Windows installed the most recent drivers available automatically.
Considering the card seems to work fine other than the hotspot shitting the bed after a day or so I'm going to assume it's just an age and/or compatibility issue with Windows

**Comment by ashvin-bhuttoo** [comment]
Feature was implemented in new version 1.0.2
Additional bug fixes and enhancements made as well.

```
v1.0.2
- Added Option to Auto Restart Hotspot at user defined interval in tray icon context menu
- PersistentHotspot will now remember the last state and resume that state after reboot
	i.e. If Hotspot was turned off, after a reboot, the hotspot will persist the Off state
- Fixed issue with PersistentHotspot randomly gets in the 'Unknown Error' state and stops working.
```
